libp2p/transports/transport.nim

```diff
@@ -99,9 +95,3 @@ method handles*(
   # having to repeat the check in every transport
   if address.protocols.isOk:
     return address.protocols.get().filterIt( it == multiCodec("p2p-circuit") ).len == 0
-
-method localAddress*(self: Transport): MultiAddress {.base, gcsafe.} =
```

Comment:
Maybe just fix it to return the address as corresponds - we can use it in the switch I believe.

Reply:
The switch just get the `transport.ma` field. Though we could add `localAddress` back and make him return `transport.ma` to allow override-ability.. but we don't need to, so I would keep it that way